admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts

```diff
@@ -222,15 +222,14 @@ export default class PositionExtension {
     rowsData: Array<RowDatas>,
   ): Array<DNDPositions> {
     const regex = /^row_(\d+)_(\d+)$/;
-    const mapping = Array(rowsData.length).map(Object);
+    const mapping = Array(rowsData.length).fill(undefined).map(Object);
 
     for (let i = 0; i < rowsData.length; i += 1) {
-      const regexResult = <RegExpPositions>regex.exec(rowsData[i].rowMarker);
 
```

Comment:
this change was added with change to TS, but position doesn't work properly with those changes so I changed it back. Would love advice how to refactor this and make it compatible with TS.